### ISSUE TITLE
Group Call: No way to get room ID.

### ISSUE THREAD

**Issue by Arsenal2321** [ciscospark/spark-android-sdk#23]
Hello,

**Is there any way to recover the ROOM_ID from a group call ?**

We are looking for a way to correlate the INCOMING CALL or ACTIVE CALL to a room. The api level does not seem to provide this information.

**Also, is there any way to know if a ROOM have an active call ongoing and also the time of the call?** It seems to be handle from the cisco application but not from the API perspective. Unless this is done through webhooks ?

**Comment by olivierproffit** [comment]
@Arsenal2321  currently only 1:1 call incoming notification works. See limitations here: https://developer.ciscospark.com/sdk-for-ios.html#incomingCallNotifications

**Comment by olivierproffit** [comment]
We plan to deliver a broader set of /call event soon. Stay tunes

**Comment by Arsenal2321** [comment]
Great. Thanks for the update

**Comment by Arsenal2321** [comment]
I have read the actual response, but my problem is not with PUSH call notification.

I receive the incoming call without any problems, but:
**I can't link which Cisco spark ROOM is linked to the current call (Incoming or active)** 

This means when I make a call to a ROOM (group call), I will never know from the call object which room is in call. I need this to display the ROOM title being called from the incoming call screen or the ROOM title in the active call screen.

**Comment by olivierproffit** [comment]
yes that's a known limitation. I can get the ID only if it's a 1:1 call as of today.

**Comment by Arsenal2321** [comment]
Ok :) Thanks for the feedback. I will wait for an api Update to integrate this.
